pyDeltaRCM/bmi_delta.py

```diff
@@ -84,109 +67,109 @@ def initialize(self, filename='deltaRCM.yaml'):
             1: 'uniform_rectilinear_grid',
             2: 'uniform_rectilinear_grid',
         }
-        
+
     def update(self):
         """Advance model by one time step."""
-        
+
         self._delta.update()
-        self._time += self.get_time_step()
-        
+        self._delta._time += self.get_time_step()
```

Comment:
are you sure this is right? Doesn't the call two lines above `self._delta.update()` already [update the timestep inside the pyDeltaRCM model?](https://github.com/ericbarefoot/pyDeltaRCM_WMT/blob/9d773e63be899cc6a6027a186c89e66c187fc027/pyDeltaRCM/deltaRCM_driver.py#L170) I think this will double update the timestep.

Reply:
You're right that's definitely a problem. I'll fix that and push.